lib/project_types/script/ui/error_handler.rb

```diff
@@ -70,7 +70,8 @@ def self.error_messages(e)
         when Layers::Domain::Errors::InvalidExtensionPointError
           {
             cause_of_error: ShopifyCli::Context.message('script.error.invalid_extension_cause', e.type),
-            help_suggestion: ShopifyCli::Context.message('script.error.invalid_extension_help'),
+            help_suggestion: ShopifyCli::Context.message('script.error.invalid_extension_help',
+             Script::Layers::Application::ExtensionPoints.types.join(', ')),
```

Comment:
Nitpick: Stylistically we usually put each arg on its own line as its a bit easier to read when things get busy. Our [ruby style guide](https://github.com/Shopify/ruby-style-guide) is pretty useful to see all the style nuances if you haven't read that yet

eg.

```ruby
ShopifyCli::Context.message(
  'script.error.invalid_extension_help',
  Script::Layers::Application::ExtensionPoints.types.join(', ')
)
```